src/components/Accordion.js

```diff
@@ -87,8 +87,14 @@ export const Accordion = React.forwardRef((props, ref) => {
     setExpanded(!isExpanded);
   };
 
+  const toggleAccordion = (open = true) => {
+    setExpanded(open);
+  };
+
   useImperativeHandle(ref, () => ({
     toggleAccordion: handleAnimation,
+    openAccordion: () => toggleAccordion(),
+    closeAccordion: () => toggleAccordion(false),
```

Comment:
@injas-bigbinary How about this approch ?
```suggestion
    toggleAccordion: handleAnimation,
    openAccordion: () => setExpanded(true),
    closeAccordion: () => setExpanded(false),
```

Reply:
@sangameshsomawar If we explicitly pass the state into `toggleAccordion` then it may cause issues if we call the function without any value just to toggle. At the moment, the function will switch the state. But if we want to pass state then the approach will be different right?